javascript/CWE-094/CodeInjectionAudit.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Audit: Usage of Code Injection sink
+ * @description Usage of Code Injection sink
+ * @kind problem
+ * @problem.severity error
+ * @security-severity 8.0
+ * @precision high
```

Comment:
Is this appropriate? It's not precise at finding _vulnerabilities_, since it's an audit query!

Reply:
Removed precision as its not needed